WcaOnRails/app/views/oauth/applications/_form.html.erb

```diff
@@ -33,7 +33,7 @@
       <%= f.text_field :scopes, class: 'form-control' %>
       <%= doorkeeper_errors_for application, :scopes %>
       <span class="help-block">
-        <%= raw t('doorkeeper.applications.help.scopes') %>
+        <%= raw t('wca.doorkeeper.applications.help.scopes') %>
```

Comment:
Oh, possibly better solution that using `raw`... can we change this to `t('wca.doorkeeper.applications.help.scopes_html')` instead?